config/clouddriver.yml

```diff
@@ -101,7 +101,9 @@ dockerRegistry:
     - name: ${providers.dockerRegistry.primaryCredentials.name}
       address: ${providers.dockerRegistry.primaryCredentials.address}
       username: ${providers.dockerRegistry.primaryCredentials.username:}
-      passwordFile: ${providers.dockerRegistry.primaryCredentials.passwordFile}
+      passwordFile: ${providers.dockerRegistry.primaryCredentials.passwordFile:}
+      repositories:
+        - ${providers.dockerRegistry.primaryCredentials.repository}
```

Comment:
What registry are you getting the entire catalog for? When I try to get the entire catalog for 'index.docker.io', I get a '401 Unauthorized' every single time.

This is at the crux of why I had trouble connecting to docker. Without these two lines, the repository specified in spinnaker-local.yml is ignored and it wasn't obvious that it was trying to get the whole catalog